.drone.yml

```diff
@@ -0,0 +1,49 @@
+kind: pipeline
+name: default
+
+services:
+- name: docker
+  image: docker:dind
+  privileged: true
+  volumes:
+  - name: dockersock
+    path: /var/run
+
+steps:
+- name: conform
+  image: autonomy/conform:c539351
+
+- name: build
+  image: autonomy/build-container:latest
+  pull: true
```

Comment:
I'm not sure, but drone docs have `pull: always`

Reply:
Oh interesting. Fixing.